tests/wallet/test_singleton.py

```diff
@@ -84,7 +84,7 @@ def test_only_one_odd_coin_created():
     try:
         cost, result = SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
     except Exception as e:
-        assert e.args == ("clvm raise",)
+        assert e.args == ("clvm raise", "80")
```

Comment:
this changed because `chia_rs` throws `ValueError` whereas `clvm_rs` throws `EvalError` (imported from the `clvm` python package). Richard made this change in order to no longer depend on the `clvm` python package (I believe)

Reply:
what's behind `80`? is that the value like `(x "80")`?

Reply:
`80` is the serialized form of an empty list or NIL, in clvm